### ISSUE TITLE
AVR 1.4.10 not in repository, is in the json

### ISSUE THREAD

**Issue by RonaldH** [adafruit/arduino-board-index#7]
package_adafruit_index.json has an entry for 1.4.10

```
 "category": "Adafruit",
          "name": "Adafruit AVR Boards",
          "url": "https://adafruit.github.io/arduino-board-index/boards/adafruit-avr-1.4.10.tar.bz2",
          "checksum": "SHA-256:9ba950a575eeb5927dc8d1849ed5dc823cbe0585d616f6cc690b5ccef4ffc1fa",
          "help": {
            "online": "https://forums.adafruit.com"
          },
          "version": "1.4.10",
          "architecture": "avr",
          "archiveFileName": "adafruit-avr-1.4.10.tar.bz2",
```

But the file isn't in the repo. The entry was added 19 hours ago.

**Comment by ladyada** [comment]
shoot! let me redo it

**Comment by ladyada** [comment]
had some tea, should be fixed? please try 1.4.11 :)

**Comment by RonaldH** [comment]
Yep, 1.4.11 works, 1.4.10 is still in the list though, but shouldn't be much of an issue.
